napari/layers/points/points.py

```diff
@@ -474,6 +1034,13 @@ def selected_data(self, selected_data):
             with self.block_update_properties():
                 self.current_size = size
 
+        properties = {
+            k: np.unique(v[index], axis=0) for k, v in self.properties.items()
+        }
+        n_unique_properties = np.array([len(v) for v in properties.values()])
+        if np.all(n_unique_properties == 1):
+            self.current_properties = properties
```

Comment:
This seems overly stringent... I suggest that we set the current properties to be the selected properties for every column where there is only one value, and ignore the other columns. ie something like:

```python
for n, prop in zip(n_unique_properties, properties):
    if n == 1:
        self.current_properties[prop] = properties[prop]
```

Reply:
I like it! I implemented your suggestion.